.github/workflows/update-catch.yaml

```diff
@@ -0,0 +1,53 @@
+name: Update Catch
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 10 1/16 * *'
```

Comment:
Hi @zoq and @rcurtin, I guess this is working fine, In the next commit I will remove `pull_request`. This cron will run the workflow at 10:00 UTC every month's 1th and 17th, we can vary this if wanted. We can also tigger this workflow according our use with the help of `workflow_dispatch`.

Reply:
The schedule approach sounds good to me.